it takes ages and on some archs it may time out

--- a/tests/isc/Makefile.am
+++ b/tests/isc/Makefile.am
@@ -43,7 +43,6 @@ check_PROGRAMS =	\
 	ratelimiter_test\
 	regex_test	\
 	result_test	\
-	rwlock_test	\
 	safe_test	\
 	siphash_test	\
 	sockaddr_test	\
